.bumpversion.cfg

```diff
@@ -0,0 +1,17 @@
+[bumpversion]
+current_version = 0.1.0-dev
+commit = True
+tag = True
+tag_name = {new_version}
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(\-(?P<release>[a-z]+))?
+serialize =
+	{major}.{minor}.{patch}-{release}
+	{major}.{minor}.{patch}
+
+[bumpversion:file:ibm_db_django/__init__.py]
```

Comment:
The version in this file should match the version or bumpversion will error.

Reply:
Doesn't this match what is in __init__.py?

Reply:
Oh sorry, I forgot that the version had been reset already.